### PR TITLE
Frosh 2020 Changes 2. Electric Boogaloo

### PR DESCRIPTION
**Pull request by JohnWu966**
Ignore the Previous Pull Request. This one is better.

Created Frosh 2020 page.

### CODE REVIEW

**Comment by modernNeo** [comment]
it seems your code has some failures. refer to https://github.com/CSSS/csss-site#before-opening-a-pr on how to fix

File: csss-site/src/events/urls.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 from django.conf.urls import url
 
 from .views import views
-from .views.frosh import frosh2012, frosh2013, frosh2014, frosh2015, frosh2016, frosh2017, frosh2018, frosh2019,frosh2020
-
+from .views.frosh import (frosh2012, frosh2013, frosh2014, frosh2015,
+                          frosh2016, frosh2017, frosh2018, frosh2019, frosh2020)
+# Seperated into two lines because the testing program doesnt like lines of code that are too long.
```
Comment (modernNeo) [inline csss-site/src/events/urls.py:6]: this comment isn't really necessary

Comment (JohnWu966) [reply csss-site/src/events/urls.py]: removed it.

File: csss-site/src/events/static/events_static/frosh/2020/landing_page/js/soon/custom.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
+/******************************************************************************************************************************
+COMMING SOON PAGE
+*******************************************************************************************************************************/
+(function ($) {
+    /**
+     * Set your date here  (YEAR, MONTH (0 for January/11 for December), DAY, HOUR, MINUTE, SECOND)
+     * according to the GMT+0 Timezone
+     **/
+    var launch = new Date(2019, 8, 3, 11, 0, 0);
```
Comment (modernNeo) [inline csss-site/src/events/static/events_static/frosh/2020/landing_page/js/soon/custom.js:9]: forgot to change the date?

File: csss-site/src/events/templates/events/frosh/2020/index.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Frosh Week 2020" />
+    <meta name="author" content="Jace Manshadi" />
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/frosh/2020/index.html:9]: I am certainly not the author

File: csss-site/src/events/templates/events/frosh/frosh_week.html [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
-    <p class="lead">Check out <a href="{%  static 'events_static/frosh/Under-Construction1.jpg' %}">The Official Frosh Week Site</a> for more info, and to register!</p>
-    <h1 class="title">The RAMing Ceremony</h1><hr>
-    <p class="lead">Our most hallowed of rituals. It doesn't hurt, we promise.</p>
-    <h1 class="title">Pac Macro</h1>
-    <hr><p class="lead">The Frosh Week classic: become Pacman himself among the streets of downtown Vancouver. Pacman and the ghosts chase each other amidst the crowds and traffic, all the while being directed by their team mates, stationed at Harbour Centre. There, they have a live overview of the positions of all players, projected on a large screen, and managed by custom CSSS software.</p>
-    <h1 class="title">Midnight Madness</h1>
-    <hr>
-    <p class="lead">An all-night event in the ASB, where the four Applied Science schools (Comp Sci, Engineering, Software Systems, and Mechtronics) go head-to-head in various challenges, until they arrive at the conclusion that everyone knew already: Comp Sci is the best.</p>
+{% extends "csss/new_header.html" %}
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/frosh/frosh_week.html:1]: this file should not have so much changed. do a re-fresh from csss master and then just change the relevant lines

File: csss-site/src/events/urls.py [hunk 5]
```diff
@@ -1,45 +1,47 @@
-from django.conf.urls import url
-
-from .views import views
-from .views.frosh import frosh2012, frosh2013, frosh2014, frosh2015, frosh2016, frosh2017, frosh2018, frosh2019
-
-urlpatterns = [
```
Comment (modernNeo) [inline csss-site/src/events/urls.py:6]: same with this file

File: csss-site/src/events/templates/events/frosh/2020/index.html [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+                <div id="timer" data-animated="FadeIn">
+                    <div id="days" class="timer_box"></div>
+                    <div id="hours" class="timer_box"></div>
+                    <div id="minutes" class="timer_box"></div>
+                    <div id="seconds" class="timer_box"></div>
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="col-lg-4 col-lg-offset-4 mt centered">
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/frosh/2020/index.html:87]: think you messed something up cause I get this error when clicking the button 
Screenshot from 2020-08-25 17-17-39

File: csss-site/src/events/templates/events/frosh/2020/index.html (hunk 3 above)
Comment (JohnWu966) [reply csss-site/src/events/templates/events/frosh/2020/index.html:9]: I only changed the Image on the page, so in a way, you kind of are.

Comment (modernNeo) [reply csss-site/src/events/templates/events/frosh/2020/index.html:9]: the name in this context is the person who updated the website for the current frosh year, which is you.

File: csss-site/src/events/templates/events/frosh/2020/frosh.html [hunk 7]
```diff
@@ -0,0 +1,331 @@
...
+
+<head>
+    <meta charset="utf-8">
+
+    <title>CSSS Frosh 2020</title>
+
+
+    <style>
+
+        body{
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/frosh/2020/frosh.html:23]: all css should go into a css file.

File: csss-site/src/events/templates/events/frosh/2020/frosh.html [hunk 8]
```diff
@@ -0,0 +1,331 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+<!-- The Following Site is heavily changed from that of previous Froshes due to the online nature of Frosh Week 2020.-->
+<!-- Go look at Frosh 2019 if you want an example of what the Frosh sites typically look like.. -->
+<!-- Contact John Wu, CSSS Frosh Chair 2020 for additional info.-->
+
+<!-- Any comments using DEBUG were for my own personal testing while designing the site. Sorry Jace. I ain't doing all that python shit to run the site the normal way. -->
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/frosh/2020/frosh.html:8]: remove all debug comments

File: csss-site/src/events/templates/events/frosh/2020/index.html [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+    <link href="{% static 'events_static/frosh/2020/landing_page/css/bootstrap.css' %}" rel="stylesheet">
+
+    <!-- Custom styles for this template -->
+    <link href="{% static 'events_static/frosh/2020/landing_page/css/soon.css' %}" rel="stylesheet">
+
+    <!-- Overrides for everything -->
+    <link href="{% static 'events_static/frosh/2020/css/overrides.css' %}" rel="stylesheet">
+
+    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!--[if lt IE 9]>
```
Comment (modernNeo) [inline csss-site/src/events/templates/events/frosh/2020/index.html:24]: you can probably remove this. doubt anyone is using IE that is so old.

**Pull request merged**